core/server/common/src/main/java/alluxio/master/journal/raft/JournalStateMachine.java

```diff
@@ -161,6 +161,13 @@ public JournalStateMachine(Map<String, RaftJournal> journals, RaftJournalSystem
     MetricsSystem.registerGaugeIfAbsent(
         MetricKey.MASTER_JOURNAL_LAST_APPLIED_COMMIT_INDEX.getName(),
         () -> mLastAppliedCommitIndex);
+    MetricsSystem.registerGaugeIfAbsent(
+        MetricKey.MASTER_JOURNAL_CHECKPOINT_WARN.getName(),
+        () -> getLastAppliedTermIndex().getIndex() - mSnapshotLastIndex
```

Comment:
if the logics are duplicated in the code base, can we use a separate method for it?

Reply:
The code is not completely duplicated. Even if a method is extracted, it is inconvenient to call it in AlluxioMasterRestServiceHandler, so I don’t think it is necessary for the time being